src/components/ComponentStatus/ComponentStatus.js

```diff
@@ -56,22 +57,24 @@ class ComponentStatus extends React.Component {
           <h2 className="page-h2">Current version: {vanillaVersion}</h2>
         </div>
         <div className="bx--col-lg-12 bx--no-gutter">
-          <table className="page-table">
-            <thead>
-              <tr>
-                <th>Component</th>
-                <th>Vanilla</th>
-                <th>React</th>
-                <th>Angular</th>
-                <th>Vue</th>
-              </tr>
-            </thead>
-            <tbody>
-              {Object.keys(componentStatus.components).map(component =>
-                this.renderItems(componentStatus.components[component])
-              )}
-            </tbody>
-          </table>
+          <TableContainer>
+            <table className="page-table">
+              <thead>
+                <tr>
+                  <th>Component</th>
+                  <th>Vanilla</th>
+                  <th>React</th>
+                  <th>Angular</th>
+                  <th>Vue</th>
+                </tr>
+              </thead>
+              <tbody>
+                {Object.keys(componentStatus.components).map(component =>
+                  this.renderItems(componentStatus.components[component])
+                )}
+              </tbody>
+            </table>
+          </TableContainer>
```

Comment:
@vpicone this is where the TableContainer component is being used

Reply:
Ah I see. We might be able to just style ComponentStatus rather than creating a component. 

We could make a ComponentStatus.module.scss and style it there.